.github/workflows/dependabot-auto-approve.yml

```diff
@@ -0,0 +1,21 @@
+name: Dependabot auto-approve updates
+on: pull_request
+
+permissions:
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+      - name: Approve a PR
+        run: gh pr review --approve "$PR_URL"
```

Comment:
Looking at the script in [spark-stats](https://github.com/CheckerNetwork/spark-stats) repository I've previously linked this script and the [script in the mentioned repository ](https://github.com/CheckerNetwork/spark-stats/blob/main/.github/workflows/dependabot-auto-approve-minor.yml) differ quite a bit. 

We would need to add a statement to only auto-approve minor version upgrades, rather than all upgrades.

Reply:
Left you a comment [here](https://github.com/CheckerNetwork/spark-stats-request-metrics/pull/9#discussion_r1977091200).